doc/nrf-bm/release_notes/release_notes_0.8.0.rst

```diff
@@ -0,0 +1,257 @@
+
+For guidelines on how to migrate an application built on nRF5 SDK to |BMshort|, refer to `Migration notes - nRF5 SDK to nRF Connect SDK Bare Metal`_.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+SDK installation
+================
+
+* The default way to install |BMshort| is through the |nRFVSC| user interface, using either the :guilabel:`Pre-packaged SDK & Toolchain` option or a GitHub installation.
+  For detailed steps, see :ref:`cloning_the_repositories_nrf_bm`.
+
+S115 SoftDevice
+===============
+
+* Updated the S115 SoftDevice version to ``s115_9.0.0-3.prototype``.
+  The SoftDevice comes in three variants to support different SoCs of the nRF54L Series: nRF54L15, nRF54L10, and nRF54L05.
+
+  * :file:`s115_9.0.0-3.prototype_nrf54l15_softdevice.hex`
+  * :file:`s115_9.0.0-3.prototype_nrf54l10_softdevice.hex`
+  * :file:`s115_9.0.0-3.prototype_nrf54l05_softdevice.hex`
+
+  For more details about this release of the SoftDevice, see :ref:`s115_docs`.
+
+  For API documentation, see `S115 9.0.0-3.prototype API reference`_.
+
+SoftDevice Handler
+==================
+
+* Added support for seeding the SoftDevice RNG when requested by the SoftDevice.
+
+* Fixed the maximum MTU Kconfig option.
+
+Boards
+======
+
+* Added the following ``bm_nrf54l15dk`` board variants for MCUboot (DFU-enabled board variants):
+
+  * ``bm_nrf54l15dk/nrf54l05/s115_softdevice/mcuboot``
+  * ``bm_nrf54l15dk/nrf54l10/s115_softdevice/mcuboot``
+  * ``bm_nrf54l15dk/nrf54l15/s115_softdevice/mcuboot``
+
+..
+
+* Renamed the ``bm_nrf54l15dk`` board variants:
+
+  * ``bm_nrf54l15dk/nrf54l05/softdevice_s115`` to ``bm_nrf54l15dk/nrf54l05/s115_softdevice``
+  * ``bm_nrf54l15dk/nrf54l10/softdevice_s115`` to ``bm_nrf54l15dk/nrf54l10/s115_softdevice``
+  * ``bm_nrf54l15dk/nrf54l15/softdevice_s115`` to ``bm_nrf54l15dk/nrf54l15/s115_softdevice``
+
+..
+
+* Fixed:
+
+  * Fixed wrong RRAM size on the nRF54L10 emulation target from 1022 to 1012 KiB.
+
+..
+
+* Removed:
+
+  * The template board variants.
+  * The ``no_softdevice`` variants of the ``bm_nRF54l15dk``.
+
+DFU
+===
+
+* DFU based on MCUboot Loader mode is now introduced in the SDK:
+
+  * Experimental support added for DFU (update of Firmware Loader, SoftDevice, and application).
+    This is an asymmetric foreground DFU bank solution - the user application bank and SofDevice are much bigger than the update Loader.
+  * This solution contains a port of the MCUmgr SMP protocol part required for performing the DFU operation.
+
+  For more details, see :ref:`bm_dfu`.
+
+Logging
+=======
+
+* Fixed an issue with the minimal log mode.
+
+Drivers
+=======
+
+* Added the following drivers:
+
+  * The :ref:`driver_lpuarte` driver.
+  * A SoftDevice-based flash driver implementing the Zephyr flash interface.
+
+Libraries
+=========
+
+* Added the following libraries:
+
+  * The :ref:`lib_bm_zms` library.
+  * The ``bm_storage`` library.
+  * A queue library ported from Zephyr.
```

Comment:
modified not ported, it's the exact same file with some lines removed, it was not ported to a new platform by recoding

Reply:
changed to
 * A queue library (modified library from Zephyr).